python/cuml/svm/svr.pyx

```diff
@@ -194,6 +194,33 @@ class SVR(SVMBase):
                                   verbose, epsilon)
         self.svmType = EPSILON_SVR
 
+    def __getstate__(self):
+        state = self.__dict__.copy()
+        del state['handle']
+        del state['_model']
+        if self.fit_once_ == 1:
+            state['dual_coef_'] = cudf.DataFrame \
```

Comment:
Can this all live in svm_base? It would be nice if we don't have to repeat it in regressor and classifier both. What is the delta required for each separate one?

Reply:
I initially wanted to avoid the reference problem we faced with Knn-regressor/classifier. But that shouldn't be an issue here - I've refactored the code.